ocfweb/docs/views/officers.py

```diff
@@ -73,7 +98,7 @@ def __str__(self) -> str:
                 s += ' (acting)'
         if not self.full_term:
             s += ' ({}–{})'.format(
-                self.start.strftime('%m/%d/%y'),
+                cast(date, self.start).strftime('%m/%d/%y'),
```

Comment:
Generally `cast`s are used a tool of last resort. For something like this, I'd recommend doing something like
```
assert self.start is not None
... # self.start should be a date from here on
since it seems that `self.start` could be `None`?

Reply:
Thanks, done.